public/app/plugins/panel/geomap/module.tsx

```diff
@@ -44,8 +44,10 @@ export const plugin = new PanelPlugin<GeomapPanelOptions>(GeomapPanel)
     if (!state?.layers) {
       // TODO? show spinner?
     } else {
+      const layersCategory = ['Map layers'];
```

Comment:
These are great changes, though I think they may be better off in a separate PR just to keep things clean (i.e. in event we need to revert for whatever reason, we could still keep our doc changes etc.)

I think we could pull out the `module.tsx` and `plugin.json` changes into a separate PR 👍 - this would also address build failing for this docs PR at least

Reply:
Removed these changes from this PR and created https://github.com/grafana/grafana/pull/53703